fw/rbcx-coprocessor/platformio.ini

```diff
@@ -36,18 +37,37 @@ extra_scripts =
 debug_tool = stlink
 upload_protocol = stlink
 
-[env:hw10]
+[env:hw10_stlink]
 build_type = debug
 build_flags =
     ${env.build_flags}
     -DRBCX_HW_VER=0x0100
     -DUSBD_DP_PORT=GPIOA
     -DUSBD_DP_PIN=8
 
-[env:hw11]
+[env:hw11_stlink]
 build_type = debug
 build_flags =
     ${env.build_flags}
     -DRBCX_HW_VER=0x0101
     -DUSBD_DP_PORT=GPIOD
     -DUSBD_DP_PIN=2
+
+# Use sboot from https://github.com/RoboticsBrno/sboot_stm32
+# Default Win10 driver does not work, use generic libusb from https://zadig.akeo.ie/
+# Usage explained at https://www.hanselman.com/blog/HowToFixDfuutilSTMWinUSBZadigBootloadersAndOtherFirmwareFlashingIssuesOnWindows.aspx
+[env:hw11_sboot]
```

Comment:
Is it possible to simply use the same sboot config for hw1.0?

Reply:
Yes, it is, but I didn't create the bootloader config for it yet (it has different USB & button(?) pins).